examples/repl.rs

```diff
@@ -20,6 +22,8 @@ mod example {
     use repl_tools::HighlightAndComplete;
     use rustyline::CompletionType;
 
+    pub type CrabResult<T> = std::result::Result<T, Box<dyn std::error::Error>>;
```

Comment:
Using `Result` here should work, right?

Reply:
you mena not the fully qualified `std::result::Result` but just Result, right ?